### DISCUSSION TITLE
Seamless textures

### DISCUSSION THREAD

**Discussion by AdrianTheFrog** [hlky/stable-diffusion-webui#1345] [Feature suggestions]
I have seen the github.com/carson-katri/dream-textures repo being shared around recently, and I think the seamless textures feature from it should be added to this one too. 
(If it is already a feature than an easily accessible checkbox to activate it would be nice). 

Seamless textures are great for 3d modelling and game development and it would be nice to have all of the features of stable diffusion accessible from the same place.

**Comment by codedealer** [comment]
It's available in Gradio UI behind `--tiling` flag currently.

**Reply by AdrianTheFrog** [reply]
A tiling checkbox under the advanced settings panel could be a good idea then.
